actions/class.ItemContent.php

```diff
@@ -205,9 +206,15 @@ public function download(): void
         $resolver = new ItemMediaResolver($item, $itemLang);
 
         $asset = $resolver->resolve($params['path']);
-        $stream = $asset->getMediaSource()->getFileStream($asset->getMediaIdentifier());
+        $mediaSource = $asset->getMediaSource();
```

Comment:
let's please add tests : ItemContentTest
also let's do a demo or use a kitchen so we can test this

Reply:
deployed to the playground, a link in the description
What kind of tests do you mean? I did not create any new public methods also actually ItemContent is endpoint controller...
Are you talking about functional end-to-end testing? Otherwise I don't see subject to test here.